src/main/java/appeng/integration/modules/NEIHelpers/NEIGuiHandler.java

```diff
@@ -0,0 +1,22 @@
+package appeng.integration.modules.NEIHelpers;
+
+import appeng.client.gui.implementations.GuiMEMonitorable;
+import codechicken.nei.api.INEIGuiAdapter;
+import net.minecraft.client.gui.inventory.GuiContainer;
+import net.minecraft.item.ItemStack;
+
+public class NEIGuiHandler extends INEIGuiAdapter {
```

Comment:
Love when somebody brings naming convention from other language and uses it inconsistently (`I` prefix means interface, in java world the practice is to append `Impl` suffux in worst case).

(Not saying you should do anything about this)